web/wdio.conf.ts

```diff
@@ -9,6 +9,39 @@ import tsconfigPaths from "vite-tsconfig-paths";
 const isProdBuild = process.env.NODE_ENV === "production";
 const apiBasePath = process.env.AK_API_BASE_PATH || "";
 const runHeadless = process.env.CI !== undefined;
+const testSafari = process.env.WDIO_TEST_SAFARI !== undefined;
+const testFirefox = process.env.WDIO_TEST_FIREFOX !== undefined;
+const skipChrome = process.env.WDIO_SKIP_CHROME !== undefined;
+
+const capabilities = [];
+
+const MAX_INSTANCES = 10;
+
+if (!skipChrome) {
```

Comment:
By default, it runs the tests in Chrome.  Environment variables now allow you to control if it should run in Firefox or Safari, and control which on-demand.